### PR TITLE
chore, refactor(egen): New template edits, clean up code refactor, remove future tense

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
The notebook `rlhf_tune_llm.ipynb` follows old template and has minor corrections. Some doc links won't work.
This PR includes the following updates:

- Manual steps involved for fetching Endpoint resource
- Adds Colab kernel restart and authentication steps
- Replace REGION with LOCATION
- Add colab enterprise link and follows new template for Colab steps
- Remove IS_TESTING
- Remove restricted documentation links (some of the links don't have public access)
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/generative_ai/rlhf_tune_llm.ipynb [hunk 1]
```diff
@@ -847,12 +835,12 @@
...
         "\n",
-        "To perform bulk inference you will need the path to your tuned third-party model, which can be found in the Vertex Pipelines UI under **Reinforcer** > **Output Parameters** > `output_model_path`. Once you have the model path, follow the directions in the [Bulk Inference notebook](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/batch_eval_llm.ipynb) to generate offline predictions from a tuned model checkpoint."
+        "To perform bulk inference you need the path to your tuned third-party model, which can be found in the Vertex Pipelines UI under **Reinforcer** > **Output Parameters** > `output_model_path`. Once you have the model path, follow the directions in the [Bulk Inference notebook](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/batch_eval_llm.ipynb) to generate offline predictions from a tuned model checkpoint."
```
Comment (kittyabs) [inline notebooks/official/generative_ai/rlhf_tune_llm.ipynb:843]: - add comma: " "To perform bulk inference, you need the path..."

- change: "Vertex Pipelines UI"
- to: "Google Cloud console for Vertex AI Pipelines"

File: notebooks/official/generative_ai/rlhf_tune_llm.ipynb [hunk 2]
```diff
@@ -946,7 +934,7 @@
         "id": "TlICJjNqvJlx"
       },
       "source": [
-        "You can also interact with your model using Generative AI Studio. Click on **Deploy Model** > **Output Parameters** > `endpoint_resource_name` hyperlink. Then click on the model name > \"Deploy & Test\" tab > \"Open in Prompt Design\"."
+        "You can also interact with your model using Generative AI Studio. Click on **Deploy Model** > **Output Parameters** > `endpoint_resource_name` hyperlink. Then click on the model name > **Deploy & Test** tab > **Open in Prompt Design**."
```
Comment (kittyabs) [inline notebooks/official/generative_ai/rlhf_tune_llm.ipynb:937]: Remove "on" in two cases:

- change: "Click on" 
- to: "Click:

- change: "click on"
- to: "click"

**Review by kittyabs — COMMENTED** [review]
Multiple edits required.

File: notebooks/official/generative_ai/rlhf_tune_llm.ipynb [hunk 3]
```diff
@@ -847,12 +835,12 @@
...
-        "The recommened way to get predictions from tuned models depends on whether you are working with a first-party or third-party model. For first-party models, e.g. `text-bison@001`, you can get predictions using [Batch Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-batch-predictions) or [Online Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-online-predictions), and for third-party models, we provide a bulk inference pipeline that can be used with the tuned-model checkpoint. Examples for both model types are in subsequent sections below.\n",
+        "The recommened way to get predictions from tuned models depends on whether you are working with a first-party or third-party model. For first-party models, e.g. `text-bison@001`, you can get predictions using [Batch Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-batch-predictions) or [Online Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-online-predictions), and for third-party models, a bulk inference pipeline is provided that can be used with the tuned-model checkpoint. Examples for both model types are in subsequent sections below.\n",
```
[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: Yes, `text-bison@001` is a PaLM model for text-generation. 
Changed the text as below

image

[the remark this answers is not in the document]
Comment (kittyabs) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: Is there a reference to the upcoming deprecation of PaLM?

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: There is no reference provided as of now. Can you please share the reference? 
Also, just found the list of supported model's reference from [here](https://cloud.google.com/vertex-ai/generative-ai/docs/models/tune-text-models-rlhf#supported_models). Removing `text-bison@001` refs.

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: Would this link work: [PaLM API deprecation](https://ai.google.dev/palm_docs/deprecation) ?

[the remark this answers is not in the document]
Comment (krishr2d2) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: I think the PaLM API deprecation [won't affect its support in Vertex AI](https://ai.google.dev/palm_docs/deprecation). This notebook describes using PaLM models through Vertex AI SDK for RLHF fine-tuning. The older model versions may not be accessible but that could be addressed by simply removing the version mention in the string.

[the remark this answers is not in the document]
Comment (kittyabs) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: https://screenshot.googleplex.com/AVczsyhpq2iEHdX. That's what appears in the generative-ai notebooks.

[the remark this answers is not in the document]
Comment (kittyabs) [reply notebooks/official/generative_ai/rlhf_tune_llm.ipynb:838]: Thanks. Looking into this to educate myself.

**Pull request merged**